Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent { docker defaultWorker.getConfig("openstack-test") }
+    options {
+        ansiColor('xterm')
+        timestamps()
+        timeout(time: 2, unit: "HOURS")
```

Comment:
If this is the timeout for one target, as of now, we could reasonably hit this timeout by adding a dozen end-to-end tests. I say we bump to 2.5 hours.